module/pych/extern.py

```diff
@@ -37,7 +37,7 @@ class Extern(object):
 
     # pylint: disable=too-many-instance-attributes
     # There are a lot but they are all reasonable.
-    def __init__(self, ename=None, lib=None, sfile=None, bfile=None, slang=None):
+    def __init__(self, depend=[], ename=None, lib=None, sfile=None, bfile=None, slang=None):
```

Comment:
Using mutable values as argument defaults usually is not what you want. A better pattern is to set the default to `None` and then in the method body use something like:

```python
if depend is None:
    depend = []
```

More info about why using mutable values as a default is not ideal: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments